dao.py

```diff
@@ -619,6 +618,25 @@ def get_user_by_session_id_or_none(self, session_id):
     def get_user_by_region(self, regions):
         pass
 
+    def get_is_superadmin(self, user_id):
+        user = None
+        if self.users_col.find_one({'_id': user_id}):
+            user = M.User.load(self.users_col.find_one({'_id': user_id}))
+            return user.admin_level == 'SUPER'
+        else:
+            return False
+
+    def set_user_admin_level(self, user_id, admin_level):
+        if type(admin_level) is not M.AdminLevels:
```

Comment:
I think M.AdminLevels does not exist anymore, so this should error (is it being tested)?
In any case, the ORM should take care of this check.